build-tools/packages/build-cli/src/commands/check/policy.ts

```diff
@@ -321,6 +313,7 @@ export class CheckPolicy extends BaseCommand<typeof CheckPolicy> {
 		const { exclusions, gitRoot, pathRegex } = commandContext;
 
 		const filePath = path.join(gitRoot, inputPath).trim().replace(/\\/g, "/");
+		assert(path.isAbsolute(filePath) === true);
```

Comment:
Should we add validations like these at the "use sites" of the file paths? E.g. if the handlers and resolvers know they need absolute paths, having them validate it seems like a good idea in case something else ends up calling them somehow.

Reply:
I like the idea, but I think it's out of scope for this change.